docker-image.el

```diff
@@ -156,6 +156,7 @@ and FLIP is a boolean to specify the sort order."
   :man-page "docker-image-run"
   :value '("-i" "-t" "--rm")
   ["Arguments"
+   ("--mount" "mount" "--mount=" read-string)
```

Comment:
One last nitpick: can you name the doc string "Mount" instead of "mount" ?

Reply:
Sure.  No worries.